usaspending_api/references/models.py

```diff
@@ -185,6 +186,18 @@ class Meta:
         db_table = 'office_agency'
 
 
+class FilterHash(models.Model):
+    filter_hash_id = models.AutoField(primary_key=True)
```

Comment:
Since you don't seem to be using `filter_hash_id` anywhere, making this change should be as simple as deleting line 190 (and regenerating the migration).  Django will automatically set up the `id`.